tests/test_workers/test_cleanup_job.py

```diff
@@ -201,6 +201,33 @@ def test_cleanup_job_success(mock_requests, mock_auth_requests, mock_dt):
     mock_requests.return_value.json.side_effect = [mock_complete, mock_in_progress, mock_failed]
     mock_auth_requests.return_value.ok = True
     main()
+    calls = [
+        mock.call(
+            "http://cachito.domain.local/api/v1/requests/50",
+            json={"state": "stale", "state_reason": "The request has expired"},
+            timeout=60,
+        ),
+        mock.call(
+            "http://cachito.domain.local/api/v1/requests/51",
+            json={"state": "stale", "state_reason": "The request has expired"},
+            timeout=60,
+        ),
+    ]
+    assert mock_requests.call_count == 3
+    assert mock_auth_requests.call_count == 2
+    mock_auth_requests.assert_has_calls(calls)
+
+
+@mock.patch("cachito.workers.config.Config.cachito_request_lifetime", 1)
+@mock.patch("cachito.workers.cleanup_job.datetime")
+@mock.patch("cachito.workers.cleanup_job.auth_session.patch")
+@mock.patch("cachito.workers.cleanup_job.session.get")
+def test_cleanup_job_fail(mock_requests, mock_auth_requests, mock_dt):
+    mock_dt.utcnow = mock.Mock(return_value=datetime(2019, 9, 13))
+    mock_dt.strptime = mock.Mock(return_value=datetime(2019, 9, 5))
```

Comment:
```suggestion
    mock_dt.strptime = datetime.strptime
```
I think it's better to use the real `strptime` method like this. The `'updated'` values in the testing data should already be set properly to make this work. Additionally, this change would make it easier to test the different lifetimes for different requests, see previous suggestion.